### DISCUSSION TITLE
ONNX model format has become 2x faster

### DISCUSSION THREAD

**Discussion by WEBPerformace** [civitai/civitai#553] [General]
ONNX has just become twice as fast as before, so I'm interested: are there any ONNX models on Civitai, and has anyone compared their performance with SafeTensors? It looks very promising.

https://devblogs.microsoft.com/directx/dml-stable-diffusion/
https://www.tomshardware.com/news/nvidia-geforce-driver-promises-doubled-stable-diffusion-performance
https://build.microsoft.com/en-US/sessions/47fe414f-97b8-4b71-ae9e-be9602713667

![image](https://user-images.githubusercontent.com/121736447/241389993-6c4b83ad-416f-4809-99f1-dd6481292cbc.png)

**Comment by rockerBOO** [comment]
ONNX container is nice because it can have wide support. In this case the optimization is for DirectML and an optimized model for DirectML.
